mono/metadata/object-internals.h

```diff
@@ -25,7 +25,7 @@
 #define MONO_CHECK_ARG(arg, expr, retval) do {				\
 	if (G_UNLIKELY (!(expr)))					\
 	{								\
-		if (arg) {} /* check if the name exists */		\
+		(void)(arg); /* check if the name exists */		\
```

Comment:
This going to conflict with #9430

Reply:
Yes, I keep wanting the same underlying library.